frontend/src/_variables.scss

```diff
@@ -1,6 +1,18 @@
 $dark: hsl(300, 1%, 14%);
+$dark-line: hsl(300, 1%, 35%);
 $navbar-dark: hsl(240, 7%, 14%);
 $light-dark: hsl(0, 0%, 60%);
 $yellow-button: hsl(43, 90%, 40%);
 $primary-yellow: hsl(65, 100%, 60%);
 $secondary-yellow: hsl(52, 100%, 50%);
+$ranking-yellow: hsl(49, 100%, 50%);
+$white-text: #ffffff;
```

Comment:
this feels a little redundant given that you can literally specify `white` in css